### PR TITLE
Pass data along to window.context

### PR DESCRIPTION
**Pull request by alanorozco**

### CODE REVIEW

File: 3p/integration.js [hunk 1]
```diff
@@ -477,7 +477,7 @@ function isAmpContextExperimentOn() {
  */
 function installContext(win) {
   if (isAmpContextExperimentOn()) {
-    installContextUsingExperimentalImpl(win);
+    installContextUsingExperimentalImpl(win, data);
```
Comment (jridgewell) [inline 3p/integration.js]: Where's `data` coming from?

Comment (alanorozco) [inline 3p/integration.js]: Module-level variable. 

BTW this PR hasn't been sent for review yet ;)

File: 3p/integration.js [hunk 2]
```diff
@@ -444,7 +444,7 @@ window.draw3p = function(opt_configCallback, opt_allowed3pTypes,
     delete data['_context'];
     manageWin(window);
     installEmbedStateListener();
-    draw3p(window, data, opt_configCallback);
+    draw3p(window, window.context.data, opt_configCallback);
```
Comment (lannka) [inline 3p/integration.js]: would you add a TODO to clean up the problematic global data object?

Comment (alanorozco) [inline 3p/integration.js]: Done

**Comment by cramforce** [comment]
Please add tests in immediate follow up.

**Pull request merged**